unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object

```diff
@@ -237,9 +237,9 @@
     </fields>
     <fields>
         <fullName>npe03__Installment_Period__c</fullName>
-        <description>How often you’ll receive the donations. NOTE: Changing the installment period for an existing Fixed-Length Recurring Donation may result in the loss and re-creation of open opportunities. Any data associated with those opportunities may be lost.</description>
+        <description>The Installment Period along with Installment Frequency defines the installment schedule of this Recurring Donation. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule is defined as &quot;every 3 months&quot;.</description>
```

Comment:
The description cannot be changed here by itself. That can only be done in the RecurringDonations package. Inline Help Text can be changed here. However, if you do change the Description in the Recurring Donations package it will need to exactly match here. It's complicated, but we're limited by how Salesforce treats metadata updates for Developer vs. Subscriber editable.

Reply:
Also, if this change to description is meant to apply only to RD2, then I don't think it should be changed at all since the field will still be used by both RD1 and RD2. There's no way to change the Description in a way that it's different for one org. It has to be identical everywhere.

Reply:
@force2b - Ok, I didn't realize that. I can revert that description change back. It does only apply to RD2.